# Contains information regarding the engagement between two people
# If either person_a or person_b are empty that signifies the robot
Header header

string person_a
string person_b
float32 distance
float32 mutual_gaze
float32 engagement
float32 confidence_a
float32 confidence_b